### PR TITLE
[popups] don't break old editor when going back from builder

### PR DESCRIPTION
**Pull request by matallo**
close https://github.com/CartoDB/cartodb/issues/8312

added the missing "none" templates, and some other details

I'm don't know why it is needed the `mustache-templates/infowindows/tooltip_light` something related to the migrator seems not OK (find trace below)

https://rollbar.com/vizzuality/CartoDB/items/19219/

also, when going back to old editor it's needed a `cartodb/table/views/tooltip/custom_templates/infowindow_light` (reported https://github.com/CartoDB/cartodb/issues/8333)

added them both and I created an issue for the later, but it's smelly at least

CR @xavijam /cc @juanignaciosl

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/components/custom-carousel/custom-carousel-item-view.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ module.exports = CoreView.extend({
         template: this.model.get('template')()
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/custom-carousel/custom-carousel-item-view.js]: This should not belong to this PR, IMHO.

**Comment by juanignaciosl** [comment]
Maybe not because of the migrator but because of vizjson 3 or any new presenter. If you think that something should be fixed in the backend let me know, please.

File: lib/assets/javascripts/cartodb3/mustache-templates/infowindows/tooltip_light.jst.mustache [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<div class="CDB-Tooltip CDB-Tooltip--isLight">
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/mustache-templates/infowindows/tooltip_light.jst.mustache]: Infowindows/tooltip_light...?

**Comment by matallo** [comment]
@juanignaciosl  what about the trace?

File: lib/assets/javascripts/cartodb3/mustache-templates/infowindows/tooltip_light.jst.mustache (hunk 2 above)
Comment (matallo) [inline lib/assets/javascripts/cartodb3/mustache-templates/infowindows/tooltip_light.jst.mustache]: > I'm don't know why it is needed the mustache-templates/infowindows/tooltip_light something related to the migrator seems not OK (find trace below)
> https://rollbar.com/vizzuality/CartoDB/items/19219/

File: lib/assets/javascripts/cartodb3/components/custom-carousel/custom-carousel-item-view.js (hunk 1 above)
Comment (matallo) [inline lib/assets/javascripts/cartodb3/components/custom-carousel/custom-carousel-item-view.js]: roger, reverted and created a new PR with them https://github.com/CartoDB/cartodb/pull/8326

**Comment by juanignaciosl** [comment]
@matallo damn, I've seen the error where backend is mixing infowindows and tooltip paths. On it. #8327

**Comment by matallo** [comment]
🚀
